server/controllers/finance/fiscal.js

```diff
@@ -83,6 +83,15 @@ function list(req, res, next) {
     JOIN user AS u ON u.id = f.user_id
   `;
 
+  const periodsSql = `
```

Comment:
Instead of changing the API for fiscal years, could we just do an additional call to `FiscalService.getPeriods()` on the client?

Reply:
The problem is : I will have to send another quest for getting periods Once I have a fiscal year. I can just write a test for the changes I made to the API

Reply:
But everyone else who requests a fiscal year will also get all the periods, even if they don't want them.  In the long run, that is probably the more expensive behavior.